### ISSUE TITLE
ion-checkbox does not toggle on click

### ISSUE THREAD

**Issue by tomsun** [driftyco/ionic#5620]
#### Short description of the problem:
`<ion-checkbox>` does not toggle when clicked

#### What behavior are you expecting?
When a (non-disabled) checkbox at http://ionicframework.com/docs/v2/components/#checkbox
is clicked in the demo view, it toggles.

**Steps to reproduce:**
1. `ionic start --v2 checkboxtest`
2. paste the checkbox snippet from the docs into page1.html, within `<ion-content>` - see below
3. `ionic serve`
4. try to toggle the checkbox

`page1.html`
```
<ion-navbar *navbar>
  <ion-title>Tab 1</ion-title>
</ion-navbar>

<ion-content padding class="page1">
  <h2>Welcome to Ionic!</h2>
  <p>
    This starter project comes with simple tabs-based layout for apps
    that are going to primarily use a Tabbed UI.
  </p>
  <p>
    Take a look at the <code>www/app/</code> directory to add or change tabs,
    update any existing page or create new pages.
  </p>

<ion-item>
  <ion-label>Daenerys Targaryen</ion-label>
  <ion-checkbox dark checked="true"></ion-checkbox>
</ion-item>

</ion-content>
```

Also attempted to wrap the ion-item/ion-checkbox block with an ion-list, like in https://github.com/driftyco/ionic-preview-app/blob/master/app/pages/checkboxes/basic/template.html - does not resolve the problem

**Ionic Version**: 2.x

**Browser & Operating System**: Safari / Firefox

```
Cordova CLI: 6.0.0
Gulp version:  CLI version 3.9.0
Gulp local:  
Ionic Version: 2.0.0-beta.1
Ionic CLI Version: 2.0.0-beta.17
Ionic App Lib Version: 2.0.0-beta.8
OS: Distributor ID:	Ubuntu Description:	Ubuntu 14.04.3 LTS 
Node Version: v4.2.6
```

**Comment by brandyscarney** [comment]
Thanks for the detailed issue! This is actually caused by the checkbox not having an `ngModel`. It was fixed in this issue #5513 which should be released in beta.2. :smile:

**Comment by tomsun** [comment]
thanks!
